sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp

```diff
@@ -1509,6 +1509,22 @@ namespace Azure { namespace Storage { namespace Test {
         blobItem.Details.RehydratePriority.Value(), Blobs::Models::RehydratePriority::Standard);
   }
 
+  TEST_F(BlockBlobClientTest, DISABLED_SetTierCold)
```

Comment:
Because the server(stg86) hasn't rolled out yet.This test case can't pass during live test.It will be enabled while stg86 rolls out.

Reply:
add an item here https://github.com/Azure/azure-sdk-for-cpp/issues/3592